develop/tutorials/articles/03-developing-a-web-application/06-permissions/01-configuring-resources.markdown

```diff
@@ -1,57 +1,55 @@
+Builder. You define all your permissions in an XML file that by convention is 
+called `default.xml` (but you could really call it whatever you want). Then you 
+implement permissions checks in the following places in your code: 
 
 - In the view layer, when showing links or buttons to protected functionality
 - In the actions, before performing a protected action
 - Later, in your service, before calling the local service
 
-First you must define your permissions scheme. You create this scheme in an XML
-file that by convention is called `default.xml` (but you could really call it
-whatever you want). Objects in your application (such as `Guestbook`s and
-`Entry`s) are defined as *resources*, and *resource actions* manage how users
-can interact with those resources. 
+You should first define the permissions you want. To get started, think of your 
+application's use cases and how access to that functionality should be 
+controlled: 
 
-You should define the permissions you want first. Look at your application and
-think of your use cases. For this app, define it like this:
+- The Add Guestbook button should be available only to administrators. 
 
-- The Add Guestbook button should be available only to administrators.
+- The Guestbook tabs should be filtered by permissions so administrators can 
+  control who can see them. 
 
-- The Guestbook tabs should be filtered by permissions in case administrators 
-  have limited who can see them.
-
-- The Add Entry button should be available only to site members to prevent
-  anonymous users from spamming the guestbook. 
+- To prevent anonymous users from spamming the guestbook, the Add Entry button 
+  should be available only to site members. 
 
 - Users should be able to set permissions on their own entries. 
 
-Now you're ready to create the permissions configuration. There are two kinds of
-permissions: portlet permissions and resource (or model) permissions. Portlet
-permissions protect access to global functions, such as *Add Entry*. If a user
-doesn't have permission to access that global function, he or she is missing a
-portlet permission. Resource permissions protect access to objects, such as
-`Guestbook`s and `Entry`s. A user may have permission to view one `Entry` and to
-view and edit another `Entry`, and may not be able to access another `Entry` at
-all. This is a resource permission. 
+Now you're ready to create the permissions configuration. Objects in your 
+application (such as `Guestbook` and `Entry`) are defined as *resources*, and 
+*resource actions* manage how users can interact with those resources. There are 
+therefore two kinds of permissions: portlet permissions and resource (or model) 
+permissions. Portlet permissions protect access to global functions, such as 
+*Add Entry*. If a user doesn't have permission to access that global function, 
+they're missing a portlet permission. Resource permissions protect access to 
```

Comment:
I used the singular "they" here to avoid the use of "he or she," which I've always thought sounded odd. But yeah, using plural "users" is the way to go if we're not using the singular "they."